src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const numOne = new Big(numberOne);
+  const numTwo = new Big(numberTwo);
+  let result;
+
+  switch (operation) {
+    case '-':
+      result = numOne.minus(numTwo);
+      break;
+    case '+':
+      result = numOne.plus(numTwo);
+      break;
+    case 'X':
+      result = numOne.times(numTwo);
+      break;
+    case '÷':
+      result = numOne.div(numTwo);
```

Comment:
- [ ] [OPTIONAL] What will the result look like the numTwo is 0 in a division?

Reply:
Thank you for the positive response @uryelah. I will work on the optional suggested change and close this Pull request.